components/core/tools/docker-images/clp-env-base-centos-stream-9/Dockerfile

```
@@ -7,6 +7,9 @@ ADD ./tools/scripts/lib_install ./tools/scripts/lib_install
 
 RUN ./tools/scripts/lib_install/centos-stream-9/install-all.sh
 
+# Install uv
+pip install uv
```

Hadolint (2.12.0)

[error] unexpected 'p'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)